src/core/router/util.js

```diff
@@ -40,6 +40,22 @@ export function stringifyQuery(obj, ignores = []) {
   return qs.length ? `?${qs.join('&')}` : '';
 }
 
+export function stripUrlExceptId(str) {
+  const [path, queryString] = str.split('?');
```

Comment:
what does this function stands for?

Reply:
To fix the links in the sidebar, only keep the id

see https://docsify-preview-git-fork-sy-records-fix-search-docsifyjs.vercel.app/preview/#/configuration?id=auto2top&f=111 and https://preview.docsifyjs.org/#/configuration?id=auto2top&f=111